backend/.pylintrc

```diff
@@ -11,24 +29,46 @@ extension-pkg-allow-list=
 # for backward compatibility.)
 extension-pkg-whitelist=
 
+# Return non-zero exit code if any of these messages/categories are detected,
+# even if score is above --fail-under value. Syntax same as enable. Messages
+# specified are enabled, while categories only check already-enabled messages.
+fail-on=
+
 # Specify a score threshold to be exceeded before program exits with error.
 fail-under=8.5
 
+# Interpret the stdin as a python script, whose filename needs to be passed as
+# the module_or_package argument.
+#from-stdin=
+
 # Files or directories to be skipped. They should be base names, not paths.
 ignore=CVS,
        migrations,
-       tests
+       tests,
+       serializers
```

Comment:
What's specific about "serializers" for them to be ignored here. They are a critical part of the implementation, it would be a shame for the linter not to cover them. If some warnings are not relevant about "serializers", shouldn't these rules be disabled in the entire project, or possibly on case by case basis with magic comments?

Reply:
If we ignore missing-class-docstring and abstract-method, then there is no need for ignoring the serializers.

Reply:
What about ml and settings?
The warnings in settings are not hard to solve and not completely pointless, but on the other side I guess it's mostly code generated by Django in the first place.
The ml has a lot of warnings that are not easy to solve, and it would be hard to get a "zero warning" policy. But there are also some warnings that may be worth solving even without activating PyLint for ml.

Reply:
About the `ml`, I agree that it should be covered by our lint tools.

The folder is currently ignored by our CI but it's a legacy configuration that we should update.

(see: https://github.com/tournesol-app/tournesol/blob/4e3b95500e5cd08a81c90a53cfe7194790897680/backend/scripts/ci/lint.sh#L15)

Enabling `pylint` in the `ml` folder can be done in a separate PR if it increases too much the scope of this one.